tests/printf_float/main.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016 Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       print floating point values test application
+ *
+ * This test is supposed to check that floating point values can be displayed
+with printf function.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include "stdlib.h"
+#include "stdio.h"
+
+int main(void)
+{
+    double floating_point_value = 2016.0311;
+    printf("Value displayed: %.2f\n", floating_point_value);
+    printf("Test successful.\n");
```

Comment:
Indeed, I initially made it a visual testing. I pushed a commit addressing your comment.